internal-services/catalog/check-embargoed-cves-task.yaml

```diff
@@ -69,6 +69,7 @@ spec:
           # workaround kinit: Invalid UID in persistent keyring name while getting default ccache
           export KRB5CCNAME=`mktemp`
           export KRB5_CONFIG=`mktemp`
+          export KRB5_TRACE=/dev/stdout
```

Comment:
I have a feeling it will break passing stdout to jq :-)
`curl --retry 3 --negotiate -u : ${OSIDB_URL}/auth/token | jq -r '.access'`

similar on other places

Reply:
Damn. Good point, thanks. I didn't realize curl would also print tons of kerberos logs as well. So probably just use `/dev/stderr` instead? Fixed.

Reply:
`/dev/stderr` should be fine

Reply:
Thanks.